src/database/migrations/20191001134213-create-user.js

```diff
@@ -8,8 +8,18 @@ module.exports = {
         primaryKey: true,
         type: Sequelize.INTEGER
       },
-      example: {
-        type: Sequelize.STRING
+      username: {
+        type: Sequelize.STRING,
+        allowNull: false,
```

Comment:
nice deliver. however, I think username should also be `unique`, you also forget to add `verified` attribute in your `user's migrations `and `user's models`. default can be `defaultValue: false`

Reply:
I don't think the username should be unique as we already have the email as unique, I don't think the user can login with his username

Reply:
yes, user can login with username, check it on other websites like `linkedin or instagram` if it is not in description you remove that attributes and add `firstName` or `lastName`

Reply:
I don't really think that's important, we can normaly mix the first and lastname in username

Reply:
Well, @lemoissonM that can make much sense now.